.devops/templates/publish-website.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  - name: version
+    type: string
+    default: 8
+
+steps:
+  - task: AzureCLI@2
+    displayName: 'Copy website drop to Azure'
+    inputs:
+      azureSubscription: 'Azure - fabricweb storage - NEW'
+      scriptType: 'bash'
+      scriptLocation: inlineScript
+      inlineScript: |
+        az storage blob upload-batch \
+        --destination 'fabric-website' \
+        --source $(System.DefaultWorkingDirectory)/apps/public-docsite/dist \
+        --account-name fabricweb \
+        --destination-path v${{ parameters.version }}/$(Build.BuildNumber)/ \
+        --auth-mode login \
+        --overwrite
+
+  - task: AzureCLI@2
+    displayName: 'Update dogfood manifest in Azure blob'
+    inputs:
+      azureSubscription: 'Azure - fabricweb storage - NEW'
+      scriptType: 'bash'
+      scriptLocation: inlineScript
+      inlineScript: |
+        az storage blob upload \
+        --account-name fabricweb \
+        --container-name fabric-website \
+        --name manifests/v${{ parameters.version }}-df.js \
+        --file $(System.DefaultWorkingDirectory)/site-manifests/v${{ parameters.version }}-df.js \
+        --auth-mode login \
+        --overwrite
```

Comment:
This step updates the manifest file for the [staging docsite](https://developer.microsoft-tst.com/fluentui) which hasn't worked in a **very** long time. I just added it here for now to have a complete step by step migration of the release tasks but open to just removing it completely